__fixtures__/test-project/web/package.json

```diff
@@ -17,14 +17,12 @@
     "@redwoodjs/forms": "0.46.1",
     "@redwoodjs/router": "0.46.1",
     "@redwoodjs/web": "0.46.1",
-    "prop-types": "15.8.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
-  },
-  "devDependencies": {
-    "autoprefixer": "^10.4.2",
+    "autoprefixer": "^9.8.8",
```

Comment:
@thedavidprice for some reason, regenerating it puts these dependencies as dependencies not devDeps... any idea why?

Reply:
I've noticed that too. Wasn't breaking anything so didn't dig in yet. But would look into:
- confirming the `yarn add` command is right
- something related to link, which doesn't seem possible
- could be yarn deduplicate... do we use that?

Weird weirdness...

Reply:
I cannot reproduce the package being moved from devDep to Dev. Running rebuildFixture works as expected on my machine.

Reply:
@dac09 the problem here was the original workaround for adding Tailwind to the test-project:
```
          execa(
           'yarn workspace web add postcss postcss-loader tailwindcss autoprefixer@^9.8.8',
            [],
            getExecaOptions(outputPath)
          ),
```

No `-D`

I'll fix it over here PR #4576

Reply:
done!